code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -202,6 +203,7 @@
 /obj/item/ammo_box/magazine/m10mm
 	name = "pistol magazine (10mm)"
 	desc = "A gun magazine."
+	icon = 'icons/tgmc/objects/ammo.dmi'
```

Comment:
You dont need to add this to every single type. This is inherited from its parent type `/obj/item/ammo_box/magazine`
```suggestion
```